about.html

```diff
@@ -0,0 +1,123 @@
+                    <li>Sponsor</li>
+                    <li>News</li>
+                    <li>LDE Campaign</li>
+                </ul>    
+            </nav>
+        </div>
+
+        <div class="intro about-intro">
+            <p class="first-para">"Ethio Art! Great Painters and Poets"</p>
+            <h1 class="first-head about-head">ART GALLERY AND POETRY READING EVENT 2023</h1>
+            <p class="second-para about-second-para">
+                The LDE event to brings together works of new and seasoned artists in Ethiopia.
+                Happening for the past ten years the event have been growing with more audience number
+                each year.
+            </p>
+            <h2 class="second-head about-second-head">You can contact us for any suggestion and questions</h2>
+            <p class="last-para about-last-para"><a href="dxrjos@gmail.com">dxrjos@gmail.com</a> </p>
+        </div>
+    </section>
+
+    <section class="logo-page">
+        <div class="logo-container">
+            <h2>Last Day Events 2023 Logo</h2>
+            <span class="underline logo-underline"></span>
+            <p class="about-logo-para">The logo of the Last Last Day Events 2023 was made by using 'Adobe Express: Free Logo Maker'.</p>
+            <div class="about-logo">
+                <img src="img/Last Day Events-logos_black.png" alt="LDE logo">
+                <p class="about-para">10 - 12 FEB 2023 | Addis Ababa</p>
+            </div>
+        </div>
+
+        <div class="logo-container simmilar-container">
+            <h2>See Past Events Organized By LDE</h2>
+            <span class="underline logo-underline"></span>
+            <p class="about-logo-para">Take a look at simmilar events organized by LDE in Addis.</p>
+            <div class="about-logo simmilar-events">
+
+                <div class="img1-cont">
+                    <img class="sim-img" src="img/past-edited.jpeg" alt="Past event 1">
+                    <p class="img1-hidden">2021 <br> Art Gallery and Wine Testing in Addis.</p>
+                </div>
+
+                <div class="img2-cont">
+                    <img class="sim-img" src="img/past-edited2.jpeg" alt="Past event 1">
+                    <p class="img1-hidden">2020 <br> Poetry, Art Gallery and Live Music in Addis.</p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="top-footer about-footer-desk">
+        <div class="footer-header">
+            <h2>Partners</h2>
+            <span class="underline logo-underline footer-underline"></span>    
+        </div>
+
+        <div class="partner-logo">
+            <img src="img/ride.png" alt="Ride Logo">
+            <img src="img/Ethiopian_Airlines_Logo.svg.png" alt="">
+            <img src="img/ayat_real_estate.png" alt="">
+        </div>
+    </section>
+
+    <footer class="bottom-footer about-bottom-footer">
+        <img src="img/Last Day Events-logos_black.png" alt="">
```

Comment:
- [ ] You have done incredibly well by using the `alt` attribute in img tag. However, you should ensure that the alt has a text. This is because alt text is critical for those with disabilities because it describes the function and appearance of a photo or graphic they cannot see. Kindly ensure to make your page more accessible by implementing the alt text 👍 .